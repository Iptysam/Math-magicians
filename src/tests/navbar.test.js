import React from 'react';
import renderer from 'react-test-renderer';
import Navbar from '../component/Pages/navbar';

it('It should render Navbar properly', () => {
  const tree = renderer
  .create(<Navbar />)
  .toJSON();
  expect(tree).toMatchSnapshot();
})
