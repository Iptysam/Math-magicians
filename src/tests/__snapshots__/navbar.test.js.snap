// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for the navigation bar `] = ` <>
    <Nav>
      <NavMenu>
        <NavLink to="src/component/Pages/Home.js" activeStyle>
          Home
        </NavLink>
        <NavLink to="/component/Calculator" activeStyle>
          Calculator
        </NavLink>
        <NavLink to="src/component/Pages/qoute.js" activeStyle>
          Qoute
        </NavLink>
      </NavMenu>
    </Nav>
  </>`;
